src/controllers/nexusController.ts

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import { gameIndexConfigs } from '../config/gameInfo.json';
```

Comment:
Consider loading this from the database in the future.

Reply:
That was in our original plan, but then I thought it would be better to have our definitions of the games we are supporting along with their IDs and names in the code rather than the database since the data in it will only change with integrating games into our code. Is this an incorrect pattern?